daemon/src/GHCSpecter/Control.hs

```diff
@@ -42,101 +42,83 @@ import GHCSpecter.UI.Types.Event
   ( BackgroundEvent (..),
     Event (..),
     ModuleGraphEvent (..),
+    MouseEvent (..),
     SessionEvent (..),
     SubModuleEvent (..),
     Tab (..),
     TimingEvent (..),
   )
 
-updateModel ::
+defaultUpdateModel ::
```

Comment:
This function is "default." 
in the end, this function will not exist as a common handler as different mode should have different event handling.